### ISSUE TITLE
Error generating coverage for net461 project ...

### ISSUE THREAD

**Issue by FlukeFan** [SteveGilham/altcover#29]
Hi,

I tried the following to get coverage for a net461 project with NUnit:

    cd MyTestProject
    dotnet new -i NUnit3.DotNetNew.Template
    dotnet new nunit --framework net461
    dotnet test

 ... outputs ...

    Total tests: 1. Passed: 1. Failed: 0. Skipped: 0.
    Test Run Successful.

Then ...

    dotnet add package AltCover
    dotnet test /p:AltCover=true

... outputs ...

    An exception occurred while invoking executor 'executor://nunit3testexecutor/': Exception has been thrown by the target of an invocation.
    Could not load file or assembly 'AltCover.Recorder.g, Version=3.5.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)

    Test Run Failed.

Let me know if there's any more information needed.  (If you miss out the **`--framework net461`** it works correctly.)

**Comment by SteveGilham** [comment]
Congratulations!  You have found not one but _two_  mistakes of mine.  

A work-around in build 580 is to do
```
dotnet build
dotnet test --no-build /p:AltCover=true /p:AltCoverAssemblyFilter=NUnit
```
The first issue is that the work-around should be
```
dotnet test /p:AltCover=true /p:AltCoverAssemblyFilter=NUnit
```
to eliminate the unwanted NUnit assembly from instrumentation, but there's a systematic error in the `AltCoverInstrumentModulesAfterBuild` target for all the filters that causes them to be ignored.

Next up is that this should have been fixed as part of issue #23, to clear out strong-names from instrumented assemblies, but clearly there's a case or set of cases I've missed.

**Comment by FlukeFan** [comment]
Hi Steve,

I can confirm the workaround works for me.

Thanks!

**Comment by SteveGilham** [comment]
Fixes for both identified issues in [pre-release build 582](https://ci.appveyor.com/project/SteveGilham/altcover/build/3.5.582-pre/artifacts).

**Comment by SteveGilham** [comment]
Should be fixed in release 3.5.587
